### DISCUSSION TITLE
Error when exporting in mkv

### DISCUSSION THREAD

**Discussion by MaxiPoivre** [nagadomi/nunif#275] [General]
Hi,
I converted a video to mkv format and I got an error : [WinError32] saying that the file can't be accessed because it's used by another process.
The convertion is finised but the file stays with the prefix _tmp_, and I have to rename it to finish the job manually.
I tried with several files and I got again the error. 
No error when exporting in mp4.

**Comment by nagadomi** [comment]
I think the cause is `the file can't be accessed because it's used by another process`.
Typically,
- The file is used by video player
- The file is used by video player on VR device via network

However, if you always get the same error result,
-  Virus checker or Thumbnailer or something is accessing the file reacting to the file closing event.

If many other people get the same error, it may be a problem on iw3 side, but at least in my windows env that error does not happen.

----

`tmp_*` is renamed after the process is complete, so the file content is the same.

**Reply by nagadomi** [reply]
I changed it that if renaming fails, wait for 2 seconds and retry 4 times.
Try to see if this change still results in an error.
